src/model/SizingSystem.cpp

```diff
@@ -1004,16 +1053,7 @@ namespace model {
     setPrecoolDesignHumidityRatio(0.008);
     setCentralCoolingDesignSupplyAirTemperature(12.8);
     setCentralHeatingDesignSupplyAirTemperature(16.7);
-    setSizingOption("NonCoincident");
-    setAllOutdoorAirinCooling("No");
-    setAllOutdoorAirinHeating("No");
     setCentralCoolingDesignSupplyAirHumidityRatio(0.0085);
-    setCentralHeatingDesignSupplyAirHumidityRatio(0.0080);
-    setCoolingDesignAirFlowMethod("DesignDay");
-    setCoolingDesignAirFlowRate(0.0);
-    setHeatingDesignAirFlowMethod("DesignDay");
-    setHeatingDesignAirFlowRate(0.0);
-    setSystemOutdoorAirMethod("ZoneSum");
```

Comment:
@joseph-robertson A bunch of these are no longer present in the Ctor, while they should. At the very least the 

    setAllOutdoorAirinHeating("No");
    setAllOutdoorAirinCooling("No");

Please put them all back.

Reply:
This is the major source of EUI deviations in the Regression tests so far

Reply:
Ok so just to recap, this is what happened:
* I commented out set methods in the ctor that were just setting to default values (i.e., unnecessary calls to set methods)
* By commenting out `setAll` methods, we have the unintended consequence of using idd default values
* We don't want to use idd defaults because it causes EUI deviations, so we'll stick with non-default values